### PR TITLE
Fix docs and README.md based on Ross's feedbacks

### PR DESCRIPTION
**Pull request by shchen-idmod**

### CODE REVIEW

File: README.md
Comment (RCarter-IDM) [inline README.md:1]: Please change to: "The recommended installation to use is:"

File: docs/cookbook/comps/ssmt.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ============
-SSMT Recipes
+SSMT recipes
 ============
 
 
-Run Analysis Remotely(Platform Analysis)
+Run analysis remotely(platform analysis)
```
Comment (RCarter-IDM) [inline docs/cookbook/comps/ssmt.rst:6]: Please add space after remotely and before left-parenthesis

File: docs/platforms/slurm/run-script-as-slurm-job.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 =======================
-Run Script As Slurm Job
+Run script as slurm job
```
Comment (RCarter-IDM) [inline docs/platforms/slurm/run-script-as-slurm-job.rst:2]: Please capitalize Slurm, as it is referring to a specific product name (or, at least appears to be the de-facto standard as such)

**Review by RCarter-IDM — APPROVED** [review]
Please change to: "The recommended installation to use is"

**Review by ZDu-IDM — APPROVED** [review]
Looks good

**Pull request merged**